remix-solidity/src/compiler/compiler-worker.ts

```diff
@@ -31,15 +27,16 @@ module.exports = function (self) {
             return JSON.stringify({ error: 'Uncaught JavaScript exception:\n' + exception })
           }
         }
-
         self.postMessage({
           cmd: 'versionLoaded',
           data: compiler.version()
         })
         break
+        
       case 'compile':
         missingInputs.length = 0
-        self.postMessage({cmd: 'compiled', job: data.job, data: compileJSON(data.input), missingInputs: missingInputs})
+        if(data.input && compileJSON)
```

Comment:
Same as above, use `{}` after a if statement if it's not inline (one line).

Reply:
This `if` is for single line